pom.xml

```
@@ -15,190 +15,127 @@
   <packaging>hpi</packaging>
 
   <name>Gitea Checks plugin</name>
-  <description>Allows users to publish Gitea commit checks</description>
+  <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
 
   <properties>
-    <java.level>8</java.level>
     <changelist>999999-SNAPSHOT</changelist>
-
-    <useBeta>true</useBeta>
-
-    <!-- Jenkins Plug-in Dependencies Versions -->
-    <checks-api.version>1.5.0</checks-api.version>
-    <plugin-util-api.version>1.7.1</plugin-util-api.version>
-    <gitea.version>1.2.1</gitea.version>
-
-    <!-- Test Library Dependencies Versions -->
-    <jetty.version>9.4.36.v20210114</jetty.version>
-    <jackson.version>2.12.1</jackson.version>
-    <wiremock.version>2.33.1</wiremock.version>
-    <guava.version>29.0-jre</guava.version>
-    <asm.version>9.1</asm.version>
-    <json-unit-core.version>2.34.0</json-unit-core.version>
-    <http-client.version>4.5.13</http-client.version>
+    <jenkins.version>2.426.3</jenkins.version>
```

```suggestion
    <jenkins.version>2.440.3</jenkins.version>
```
as you're referencing `bom-2.440.x` later in the POM.

---

this doesn't fix the tests. I like to support the older Jenkins version if possible for first release and then bump the Jenkins version again later

---

I see. 😉